src/scf-release/packages/helm/packaging

```diff
@@ -4,26 +4,15 @@ set -e
 
 BIN_DIR="${BOSH_INSTALL_TARGET}/bin"
 
-VERSIONS=""
-VERSIONS="${VERSIONS} 2.11.0"
-VERSIONS="${VERSIONS} 2.8.2"
+VERSION="2.11.0"
```

Comment:
We should still pack a few versions so that the happy path is not installing extra things

Reply:
I would be more happy with that as well.

Reply:
Done! I'm also happy with it!